.github/pull_request_template.md

```diff
@@ -1,9 +1,11 @@
-<!-- Add a small description here of the changes you added -->
+## Edited
+<!-- Add a brief description here -->
```

Comment:
It will be interesting to see how people use these two sections now... Very happy to try this out...